### PR TITLE
get list of events

### PR DESCRIPTION
**Pull request by yogesh-pawar-2023**

### CODE REVIEW

File: src/src/beneficiaries/beneficiaries.service.ts [hunk 1]
```diff
@@ -93,20 +93,29 @@ export class BeneficiariesService {
...
+   const sortType=body?.sortType ? body?.sortType : 'desc'
+    const page = body?.page ? body?.page : '1';
     const limit = body?.limit ? body?.limit : '10';
-
     let offset = 0;
     if (page > 1 && limit) {
       offset = parseInt(limit) * (page - 1);
     }
     let query = '';
     if (status) {
```
Comment (sagarkoshti1990) [inline src/src/beneficiaries/beneficiaries.service.ts:107]: check `if(status && status !== "")`

File: src/src/beneficiaries/beneficiaries.service.ts [hunk 2]
```diff
@@ -93,20 +93,29 @@ export class BeneficiariesService {
...
-      let query = `{program_beneficiaries:{status:{_eq:${status}}}}`;
+         query = `{program_beneficiaries:{status:{_eq:${status}}}}`;
+    }
+    let search='';
+
+    if(body.search){
```
Comment (sagarkoshti1990) [inline src/src/beneficiaries/beneficiaries.service.ts:112]: `if(body.search && body.search !== "")`

File: src/src/beneficiaries/beneficiaries.service.ts (hunk 1 above)
Comment (yogesh-pawar-2023) [reply src/src/beneficiaries/beneficiaries.service.ts:107]: update as per comment

File: src/src/beneficiaries/beneficiaries.service.ts (hunk 2 above)
Comment (yogesh-pawar-2023) [reply src/src/beneficiaries/beneficiaries.service.ts:112]: update as per comment

**Pull request merged**